### PR TITLE
[REACTOS] Spelling fixes

### PR DESCRIPTION
**Pull request by ace-dent**
## Purpose

Fix spellings for consistency in source code.
Replaces PR #5503 with a narrower scope. Checked it doesn't affect third-party files [(1)](https://github.com/reactos/reactos/blob/master/media/doc/3rd%20Party%20Files.txt),[(2)](https://github.com/reactos/reactos/blob/master/media/doc/WINESYNC.txt).

## Proposed changes

Fixes:
- Unassociate.
- Read-only.
- Double-click.

### CODE REVIEW

**Comment by ace-dent** [comment]
@binarymaster @JoachimHenze for review.

**Review by learn-more — APPROVED** [review]

**Review by JoachimHenze — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]
I'm not sure rdesktop in rosapps and wdf code is forked, but otherwise fine.

File: base/shell/cmd/lang/en-US.rc [hunk 1]
```diff
@@ -121,7 +121,7 @@ ERASE [/N /P /T /Q /S /W /Y /Z /A[[:]attributes]] file ...\n\n\
   /S    Delete file from all sub directory\n\
   /A    Select files to be deleted based on attributes.\n\
         attributes\n\
-        R     Read Only files\n\
+        R     Read-only files\n\
```
Comment (HBelusca) [inline base/shell/cmd/lang/en-US.rc:124]: Since you fixed this "Read only <something>", you may double-check the following list:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=read+only+&sr=1
It seems there are a bit more files that may need a similar fix.

**Review by HBelusca — APPROVED** [review]

**Review by HBelusca — CHANGES_REQUESTED** [review]
> I'm not sure rdesktop in rosapps and wdf code is forked, but otherwise fine.

WDF is 3rd-party (from open-source MS github).

File: base/shell/cmd/lang/en-US.rc (hunk 1 above)
Comment (ace-dent) [reply base/shell/cmd/lang/en-US.rc:124]: Thanks @HBelusca for the review.
Depending on context, it may or may not need a hyphen:
- ‘Read-only file selected’ (Adjective - computing).
- ‘Read only the first line’ (Verb + Adverb)

I don't have time to look at this currently. Perhaps an \#EasyHack to note somewhere?

**Comment by ace-dent** [comment]
Thanks @binarymaster @HBelusca for considering 3rd part code.
Please could you make sure / update the 'official' 3rd part lists for newcomers... [(1)](https://github.com/reactos/reactos/blob/master/media/doc/3rd%20Party%20Files.txt),[(2)](https://github.com/reactos/reactos/blob/master/media/doc/WINESYNC.txt).

`rdesktop` it looks like a few years out of sync with upstream for commits... safe to assume this is a ROS fork?
`wdf` couldn't see on the lists. Does have MS Copyright... not sure there's any harm in merging this as is?

**Comment by HBelusca** [comment]
> `wdf` couldn't see on the lists. Does have MS Copyright... not sure there's any harm in merging this as is?

3rd-party doc file updated at https://github.com/reactos/reactos/pull/5539

File: sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp [hunk 2]
```diff
@@ -319,7 +319,7 @@ class FxObject {
     union {
         //
         // This field is set when the object is being contructed or before
-        // Commit() and from then on is read only.
+        // Commit() and from then on is read-only.
```
Comment (HBelusca) [inline sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp:322]:
```suggestion
        // Commit() and from then on is read only.
```

Comment (ace-dent) [reply sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp:322]: I think here 'read-only' is an adjective, so the PR commit is correct....?

Comment (HBelusca) [reply sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp]: yes exactly, because 3rd-party :D

File: modules/rosapps/applications/net/tsclient/rdesktop/uiports/qtewin.cpp [hunk 3]
```diff
@@ -922,7 +922,7 @@ void QMyMainWindow::MemuClicked(int MenuID)
     rdp_send_input(0, RDP_INPUT_SCANCODE, RDP_KEYRELEASE, 0x1d, 0); // control
     rdp_send_input(0, RDP_INPUT_SCANCODE, RDP_KEYRELEASE, 0x38, 0); // alt
   }
-  else if (MenuID == 4) // double click
+  else if (MenuID == 4) // double-click
```
Comment (HBelusca) [inline modules/rosapps/applications/net/tsclient/rdesktop/uiports/qtewin.cpp:925]: That one as well is 3rd-party, you may revert this change.

**Comment by ace-dent** [comment]
Thanks again @HBelusca for your help. Hopefully all good now!
I think the path for `rdesktop` may not be correct [here](https://github.com/reactos/reactos/blob/e4c48945643460fca273217b75f3e7b72bb3ed7a/media/doc/3rd%20Party%20Files.txt#L10)...?

**Review by HBelusca — APPROVED** [review]

**Comment by HBelusca** [comment]
Thanks!
